msteams-platform/concepts/design/design-teams-app-fundamentals.md

```diff
@@ -81,7 +81,7 @@ Quickly learn about the fundamentals of Teams app design. You can find comprehen
    :::column-end:::
    :::column span="1":::
 
-      :::image type="content" source="../../assets/images/design-guidelines/teams-typography.png" alt-text="Conceptual image of Teams typography.":::
+      :::image type="content" source="../../assets/images/design-guidelines/teams-typography.png" alt-text="The conceptual image of Teams typography.":::
```

Comment:
May not be a good alt-text. Please get it peer reviewed.

Reply:
resolved